src/main/kotlin/no/nav/syfo/personstatus/PersonOversiktStatusHendelseAksesseringQueries.kt

```diff
@@ -60,6 +66,19 @@ fun Connection.createPersonOversiktStatus(
         } else {
             it.setNull(11, NULL)
         }
+        it.setObject(12, personOversiktStatus.latestOppfolgingstilfelle?.oppfolgingstilfelleUpdatedAt)
+        it.setObject(13, personOversiktStatus.latestOppfolgingstilfelle?.oppfolgingstilfelleGeneratedAt)
+        it.setObject(14, personOversiktStatus.latestOppfolgingstilfelle?.oppfolgingstilfelleStart)
+        it.setObject(15, personOversiktStatus.latestOppfolgingstilfelle?.oppfolgingstilfelleEnd)
+        if (personOversiktStatus.latestOppfolgingstilfelle != null) {
+            it.setString(
+                16,
+                personOversiktStatus.latestOppfolgingstilfelle.oppfolgingstilfelleBitReferanseUuid.toString()
+            )
+        } else {
+            it.setNull(16, NULL)
```

Comment:
Mulig dette virker, men tror det riktigste er å skrive it.setNull(16, Types.VARCHAR)

Reply:
Endret til `it.setNull(16, Types.CHAR) nå`.